tests/docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk:8
+
+MAINTAINER Raghav Kumar Gautam
+# commands to update docker image:
+#   - docker build . -t raghavgautam/kfk-image
+#   - docker push raghavgautam/kfk-image
```

Comment:
I suggest setting "MAINTAINER" to "Apache Kafka <dev@kafka.apache.org>".  This is similar to an \@author tag in the code... it should not be set to an individual person's name, for the same reason we don't have \@author tags.

We should probably leave out the "commands to update docker images" for now.  This would really need to be part of our official release process in order to be useful.  As it is, nobody is going to be able to upload to raghavgautam/kfk-image except raghavgautam (I assume, unless there is a security hole?)  Eventually we can discuss how to secure our official kafka images (we need a strategy for managing the private keys to the docker registry and using them as part of the Kafka release)